lang/java/reef-applications/reef-vortex/pom.xml

```diff
@@ -28,4 +28,140 @@ under the License.
         <artifactId>reef-applications</artifactId>
         <version>0.12.0-incubating-SNAPSHOT</version>
     </parent>
+
+    <dependencies>
```

Comment:
Changed to `vortex`, following how wake(and tang) is just `wake`, not `reef-wake`.